eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Translation.hs

```diff
@@ -115,13 +110,13 @@ instance
       SNothing -> pure SNothing
       SJust axd -> SJust <$> translateViaCBORAnn "auxiliarydata" axd
     let validating = IsValid True
-    pure $ Tx $ ValidatedTx bdy txwits validating aux
+    pure $ Tx $ AlonzoTx bdy txwits validating aux
 
 --------------------------------------------------------------------------------
 -- Auxiliary instances and functions
 --------------------------------------------------------------------------------
 
-instance (Crypto c, Functor f) => TranslateEra (AlonzoEra c) (API.PParams' f)
+instance (Crypto c, Functor f) => TranslateEra (AlonzoEra c) (API.ShelleyPParamsHKD f)
```

Comment:
It's weird that `ShelleyPParamsHKD` is in the `API`... perhaps a cleanup for another day: make sure that things in the API are not shelley specific.

Reply:
Yeah, I haven't looked into the `TranslateEra` class yet at all, I am sure we make some improvements there as well.

Reply:
Oh, I just realized what you meant. The fact that `API.ShelleyPParamsHKD` is reexported from `API`. Yes, I totally agree, that is not right. Again, API needs to redesigned and it should be a standalone package, not something baked into Shelley